server/routes/allocate-key-workers/view.njk

```diff
@@ -0,0 +1,159 @@
+{% extends "partials/layout.njk" %}
+{% from "moj/components/date-picker/macro.njk" import mojDatePicker %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "partials/mojPagination/macro.njk" import mojPagination %}
+
+{% set fullWidth = true %}
+{% set pageTitle = "Allocate key workers to prisoners" %}
+
+{% block innerContent %}
+  <h1 class="govuk-heading-l govuk-!-margin-bottom-6">{{ pageTitle }}</h1>
+
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+
+      <div class="govuk-grid-column-three-quarters">
+        <form method="post" class="govuk-grid-row govuk-!-margin-bottom-7 govuk-!-padding-6 search-form">
+          <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+          <h2 class="govuk-heading-m">Filter by</h2>
+          <div class="horizontal-form">
```

Comment:
Alex was quite specific about the layout and width of some of these inputs. If you haven't already, would you mind demoing the current to him to check he's happy with your interpretation of the design? I also think it would be worth checking the same filter on manage key workers to confirm the implementation and classes used matches